qutebrowser/test/mainwindow/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""
+pytest fixtures and utilities for testing.
+
+Fixtures defined here will be visible to all test files in this directory and
+below.
+"""
+
+import pytest
+
+from qutebrowser.config.config import ConfigManager
+from qutebrowser.utils import objreg
+
+
+@pytest.yield_fixture
+def default_config():
+    """
+    Fixture that registers an empty config object into the objreg module.
+
+    Should be used by tests which create widgets that obtain their initial state
+    from the global config object.
+
+    Note:
+
+        If we declare this fixture like this:
+
+            @pytest.yield_fixture(autouse=True)
+
+        Then all tests below this file will have a default config registered
+        and ready for use. Is that desirable?
```

Comment:
I have to think about that. Currently I have tests where I just have a small fake config with just the necessary things set - that'd still be possible using `config.set(...)` then, but that also tests a lot of code not related to each test.

So maybe it might be more desirable to have a fake config fixture which either is initialized with all data from `qutebrowser.config.configdata`, or is "configured" with each test to explicitely hold the options/values a given test needs?